### ISSUE TITLE
No secret after install on 6.2

### ISSUE THREAD

**Issue by pgampe** [pgampe/authenticator#4]
No secret is shown in the user settings. Saving the settings once fixes the problem.

**Comment by safersurfing** [comment]
Ok did what you suggested: Please disable 2 factor again (either via database be_users.tx_authenticator_enabled or via another user). Then safe your user settings once before activating the QR code.

Hower still the QR code does not show up. Where exactly should it show up.

![2](https://cloud.githubusercontent.com/assets/16236460/20238079/2edd9c74-a8e3-11e6-8f09-58a6a2b5fd57.PNG)

**Comment by pgampe** [comment]
You do not activate the QR code, but you active the two factor authentication with TOTP.

This is how it should look like for a freshly created user. Make sure that it actually shows the key and the QR code. Also check for possible JS errors in the backend as the QR code is generated via Javascript.

![image](https://cloud.githubusercontent.com/assets/855238/20238120/18feba86-a8e4-11e6-869b-f4b3aa047621.png)

**Comment by pgampe** [comment]
You might need to check the extension configuration (Extension Manager) to enable showing the QR code in the backend user settings. It should be enabled by default though.

**Comment by pgampe** [comment]
Can you check whether this was fixed with the last commit to master?

**Comment by taalas** [comment]
After installing the latest version in the TER I don't see any QR codes in the backend either. (TYPO3 7)

**Comment by pgampe** [comment]
@taalas TER is not master. Please save the backend user settings once and the open it again. Then it should work with the TER version.

**Comment by taalas** [comment]
@pgampe No, saving and reopening does not change the look of the backend form. I will gladly pull the latest version from the repo though and try again.

**Comment by taalas** [comment]
Issue was resolved. I was using the backend user record instead of the user settings.

**Comment by cytec** [comment]
@pgampe this is kind of confusing... so i'll have to create a new user in user records, switch to user settings for that user and activate it there. Shouldn't it be possible to do the same thing in user records? needed a while to figure this out, since the "Use two factor auth" checkbox is indeed available in the user config's advanced tab but doesn't show the code when checked...

**Comment by pgampe** [comment]
@cytec correct, the token is a personal **secret**. The checkbox is there to easily disable this setting for a user.
